### PR TITLE
Add release branch as an input

### PR DESCRIPTION
**Pull request by adamshortev**

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-name: 'Version Increment'
+name: 'Version Increment (Octo)'
```
Comment (oev-chrisbennett) [inline action.yml:2]:
```suggestion
name: 'Version Increment'
```

File: README.md [hunk 2]
```diff
@@ -1,3 +1,6 @@
+Forked from https://github.com/reecetech/version-increment. Likely only here until additional changes made in this fork are accepted back in the main repository.
+
+
```
Comment (oev-chrisbennett) [inline README.md:3]:
```suggestion
```

**Review by oev-chrisbennett — COMMENTED** [review]
👋 Just some minor edits to ensure it can be merged without editing!

**Comment by adamshortev** [comment]
@oev-chrisbennett cheers, forgot they'd show up here!

**Comment by ps-jay** [comment]
Cheers for the suggestion, I'll review in the next few days

File: action.yml [hunk 3]
```diff
@@ -22,6 +22,10 @@ inputs:
       Not applicable to `calver` scheme
     required: false
     default: 'patch'
+  release_branch:
+    description: "Specify a non-main branch to use for the release tag (the one without -prerelease). If empty, will use the default branch for the repo."
+    required:
```
Comment (oev-chrisbennett) [inline action.yml:27]: `required: false`?

**Comment by jaysin586** [comment]
Is there an ETA on this?

**Comment by ps-jay** [comment]
Sorry!  I forgot to put this on my TODO list .. and so it didn't get done 😞 

I've raised a PR to update this PR: https://github.com/octoev/version-increment-octo/pull/1

**Comment by ps-jay** [comment]
To avoid people waiting any longer for this functionality, I've decided I'll merge this, and immediately follow it up with #23

**Review by ps-jay — APPROVED** [review]

**Pull request merged**